tensorly/regression/tests/test_cp_plsr.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+
+from ..cp_plsr import CP_PLSR
+from ...base import tensor_to_vec, partial_tensor_to_vec
+from ...metrics.regression import RMSE
+from ... import backend as T
+from ...testing import assert_
+
+
+def test_CPRegressor():
+    """Test for CP_PLSR."""
+
+    # Parameter of the experiment
+    image_height = 8
+    image_width = 8
+    n_channels = 3
+    tol = 0.05
+
+    # Generate random samples
+    rng = T.check_random_state(1234)
+    X = T.tensor(
```

Comment:
Would it be better here to use `random_cp` to ensure `X` has a CP structure to it?

Reply:
Or random_tucker...